package.json

```diff
@@ -74,8 +74,9 @@
     "@changesets/apply-release-plan": "6.0.0",
     "graphql": "16.4.0",
     "cross-undici-fetch": "0.4.2",
-    "@types/react": "18.0.1",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "@types/react": "17.0.39",
+    "@types/react-dom": "17.0.11",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```

Comment:
why provide it in root package.json?

Reply:
because I don't want 900000 different versions of react installed by those packages deep in the tree that fuck up declaring loose enough dependency ranges and cause multiple instances of react runtime issues.